polaris-react/src/components/Checkbox/Checkbox.scss

```
@@ -142,13 +142,12 @@
   width: 100%;
   height: 100%;
   // stylelint-disable-next-line -- generated by polaris-migrator DO NOT COPY
-  @include focus-ring($border-width: var(--p-border-width-2));
+  @include focus-ring($border-width: var(--p-border-width-0165));
```

```suggestion
  @include focus-ring($border-width: var(--p-border-width-050));
```

`--p-border-width-2` should be replaced with `--p-border-width-050` here.